doc/schemas/sendcustommsg.request.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "required": [
+    "node_id",
+    "msg"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "node_id": {
+      "type": "pubkey",
+      "added": "v23.02"
+    },
+    "msg": {
+      "type": "hex",
+      "added": "v23.02"
```

Comment:
These args were not added, the entire command was, so "added" should really go there.

(We don't use this info yet, but we will!)

And it was added back in v0.10.1.  I'll fix...